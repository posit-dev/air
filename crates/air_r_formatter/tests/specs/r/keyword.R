return
break
next
TRUE
FALSE
NULL
Inf
NaN
NA
NA_integer_
NA_real_
NA_complex_
NA_character_
