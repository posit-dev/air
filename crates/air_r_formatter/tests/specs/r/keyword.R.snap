---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/keyword.R
---
# Input

```R
return
break
next
TRUE
FALSE
NULL
Inf
NaN
NA
NA_integer_
NA_real_
NA_complex_
NA_character_

```


=============================

# Outputs

## Output 1

-----
Indent style: Space
Indent width: 2
Line ending: LF
Line width: 80
Persistent line breaks: Respect
Table: fcase, tribble
-----

```R
return
break
next
TRUE
FALSE
NULL
Inf
NaN
NA
NA_integer_
NA_real_
NA_complex_
NA_character_
```
